### PR TITLE
[ML] Perform artifact uploads from BuildKite only

### PR DESCRIPTION
**Pull request by edsavage**
* Disable uploading from Jenkins
* Enable uploading from BuildKite

### CODE REVIEW

**Comment by edsavage** [comment]
BuildKite CI failures are due to an issue with Rosetta on the native macOS aarch64 worker.

**Comment by edsavage** [comment]
buildkite retest this please

File: dev-tools/jenkins_ci.sh [hunk 1]
```diff
@@ -159,9 +159,9 @@ case `uname` in
         exit 4
         ;;
 esac
-
-# If this isn't a PR build and isn't a debug build then upload the artifacts
-if [[ -z "$PR_AUTHOR" && -z "$ML_DEBUG" ]] ; then
-    (cd .. && ./gradlew --info -Dbuild.version_qualifier=$VERSION_QUALIFIER -Dbuild.snapshot=$BUILD_SNAPSHOT upload)
-fi
+# Do not upload artifacts from the Jenkins' build while testing uploading from BuildKite pipelines
```
Comment (droberts195) [inline dev-tools/jenkins_ci.sh:162]: Something similar needs doing in `dev-tools/jenkins_ci.ps1`.

And as a result of this change one more change will also be needed to the Jenkins builds.

Please add an `exit 0` and a comment saying why near the beginning of `dev-tools/jenkins_combine_artifacts.sh`. This won't matter for PR builds, but will stop the Jenkins snapshot and staging builds from failing due to missing artifacts, or from fighting with the BuildKite builds.

**Review by droberts195 — APPROVED** [review]
LGTM if CI is green

**Pull request merged**